cdap-watchdog/src/main/resources/cdap-log-pipeline.xml

```diff
@@ -21,7 +21,7 @@
   <!-- Shouldn't change -->
   <contextName>cdap</contextName>
 
-  <!-- This is for backward compatibility for <= 4.0, since it was hardcoded to Bytes.toBytes(100) -->
+  <!-- This is for backward compatibility for <= 4.0, since it was hardcoded to Bytes.toBytes(100). Don't modify. -->
   <property name="log.process.pipeline.checkpoint.prefix.num" value="100"/>
```

Comment:
also till 4.0, we had separate checkpoint management for cdap kafka reader plugin and log metrics plugin. while now the cdap system pipeline (both metrics and log processor) will use the same offset. there might be some duplicate/missing metrics based on offset difference after upgrade. its not critical, we can just document this.